doc/commands.md

```diff
@@ -101,6 +101,30 @@ Exposition obeys the following rules: inside a module,
 - externally defined protected symbols cannot appear in the right hand side of a
   rewriting rule
 
+**Sequential rules:**
```

Comment:
I would prefer to put this in modifiers:
sequential: By default, the order of rule declarations is not taken into account. This modifier tells Lambdapi to apply rules defining a sequential symbol in the order they have been declared (note that this may depend on the order of require commands). WARNING: use with caution as this may break important properties.

Reply:
Note that this may break confluence and other properties on terms with metavariables when rules are non linear like in your example. Take for instance the term `E=mem ?a (cons ?b nil)`. If at time t=0, ?a and ?b are different, E reduces to `false`. If, later, at time t=1, ?a is instantiated to ?b for some reason, then the application of the rule was wrong...

Reply:
I have slightly reworded, and have changed a bit the doc.